src/tests/fuzztests/xi_fuzztest_mqtt_layer.cpp

```diff
@@ -0,0 +1,10 @@
+#include <memory.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <string.h>
+
+extern "C" int LLVMFuzzerTestOneInput( const uint8_t* data, size_t size )
```

Comment:
might want to put a comment here saying "This is the fuzzer signature, and we cannot change it" or some such.